README.md

```diff
@@ -2,11 +2,75 @@
 
 This repository contains the Dockerfile for creating the caleydoapp.org landing page nginx server. In addition, this server also proxies to the various caleydoapp.org pages based on environment variables.
 
+A Docker image is available at [Docker Hub](https://hub.docker.com/r/caleydo/phovea_landing_page).
+
+## Configuration
+
+The ngnix server can be configured using the following environment variables (ENV):
+
+* [`PHOVEA_ENABLE_SSL_LANDING_PAGE`](#enable-ssl-for-landing-page)
+* [`PHOVEA_APP_*`](#add-phovea-app)
+* [`PHOVEA_APPFORWARD_*`](#add-phovea-app-forward)
+* [`PHOVEA_FORWARD_*`](#add-caleydo-subdomain-forward)
+
+
+### Enable SSL for landing page
+
+If the variable `PHOVEA_ENABLE_SSL_LANDING_PAGE` exists a SSL certificate for caleydoapp.org is added. By default no SSL certificate is added.
+
+
+### Add phovea app
+
+The variable `PHOVEA_APP_*` adds an app to the landing page and creates a ngnix configuration and a SSL certificate for this app automatically.
+
+`PHOVEA_APP_<NAME>=<name>;<domain>;<forward>;<release-channel>`
+
+* `<name>`: Name of the app that is visible in the landing page
+* `<domain>`: Subdomain of the app only (e.g., `lineup`)
+* `<forward>`: Target requests are forwarded to (e.g., `cluster:12345`)
+* `<release-channel>`: Release channel can be `stable`, `daily`, or `development` and is indicated on the landing page
+
+Example: `PHOVEA_APP_LINEUP=LineUp;lineup;cluster:12345;stable`
+
+
+### Add phovea app forward
+
+The variable `PHOVEA_APPFORWARD_*` adds an app to the landing page only. No ngnix configuration and SSL certificate is created.
+
+`PHOVEA_APPFORWARD_<NAME>=<name>;<domain>;<forward>;<release-channel>`
```

Comment:
what does it do then? or whom is handling it?

Reply:
In this case only the value of the env variable will be written into the app.csv file (see https://github.com/phovea/phovea_landing_page/pull/8#diff-c699f3c0448325c3edb34206d3d587faR21). The app.csv is then used to generate the app list on the landing page.

Does this answer your question?